schemes/scheme-model.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../data/db-config.js');
+
+module.exports = {
+    find, 
+    findById, 
+    findSteps, 
+    add, 
+    update, 
+    remove
+}
+
+function find() {
+    return db('schemes')
+}
+
+function findById(id) {
+    return db('schemes')
+    .where({id})
+}
```

Comment:
You might try to combine the top two functions here. You can have a condition that looks for the passed id. If you don't have it, it'll return the find statement there...if you do, it'll return the findById return statement.